frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -83,12 +84,18 @@ export const KnativeResourceOverviewPage: React.ComponentType<KnativeResourceOve
     );
   }
 
+  const {
+    metadata: { labels },
+  } = item?.obj;
+  const isServerlessFunction = labels && labels['boson.dev/function'];
```

Comment:
```suggestion
  const isServerlessFunction = labels?.['boson.dev/function'];
```
and better put label `boson.dev/function` in a const as it has been used multiple places.

Reply:
done